cloudplatform/connectivity-destination-service/src/test/java/com/sap/cloud/sdk/cloudplatform/connectivity/DestinationServiceTest.java

```diff
@@ -1830,6 +1830,26 @@ public void performanceTest()
         assertThat(httpClient).isSameAs(HttpClientAccessor.getHttpClient(destination));
     }
 
+    @Test
+    @DisplayName( "Custom headers in DestinationOptions are forwarded to Destination Service" )
+    void testCustomHeadersForwardedToDestinationService()
+    {
+        final Header h1 = new Header("X-Custom-1", "value-1");
+        final Header h2 = new Header("X-Custom-2", "value-2");
```

Comment:
```suggestion
        final Header h1 = new Header("X-Custom-Header-1", "value-1");
        final Header h2 = new Header("X-Custom-Header-2", "value-2");
```

Just to make it consistent to the test in `DestinationServiceAdapterTests`